src/api/routes/auth/logout.ts

```diff
@@ -0,0 +1,16 @@
+import { route } from "@fosscord/api";
+import { Request, Response, Router } from "express";
+
+const router: Router = Router();
+export default router;
+
+router.post("/", route({}), async (req: Request, res: Response) => {
+	if (req.body.provider != null || req.body.voip_provider != null) {
```

Comment:
Schema?

Reply:
Not required, since input isn't used for anything besides being logged to console